### PR TITLE
remove postup/down

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

File: nmproxy/router/iptables_linux.go [hunk 1]
```diff
@@ -25,11 +25,10 @@ const (
 )
 
 type iptablesManager struct {
-	ipv4Client   *iptables.IPTables
-	ipv6Client   *iptables.IPTables
-	ingRules     serverrulestable
-	defaultRules ruletable
```
Comment (Aceix) [inline nmproxy/router/iptables_linux.go:31]: Did you want to remove this defaulRules too?

Comment (mattkasun) [reply nmproxy/router/iptables_linux.go:31]: I only deleted that to fix a staticcheck.

**Review by Aceix — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
postup and postdown is gone from config files.
pass

**Comment by 0xdcarns** [comment]
merge conflicts require resolution

File: nmproxy/router/iptables_linux.go [hunk 2]
```diff
@@ -25,11 +25,10 @@ const (
 )
 
 type iptablesManager struct {
-	ipv4Client   *iptables.IPTables
```
Comment (0xdcarns) [inline nmproxy/router/iptables_linux.go:28]: see comment in PR

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Review by 0xdcarns — APPROVED** [review]

**Pull request merged**